pkg/storage/repodb/sync_repodb.go

```diff
@@ -0,0 +1,274 @@
+package repodb
+
+import (
+	"encoding/json"
+	"errors"
+
+	godigest "github.com/opencontainers/go-digest"
+	ispec "github.com/opencontainers/image-spec/specs-go/v1"
+
+	zerr "zotregistry.io/zot/errors"
+	"zotregistry.io/zot/pkg/log"
+	"zotregistry.io/zot/pkg/storage"
+)
+
+func SyncRepoDB(repoDB RepoDB, storeController storage.StoreController, log log.Logger) error {
```

Comment:
Comments on the methods pls. What is code in this file doing?

Reply:
Will all comment. The code in this file is responsible for reading existing oci layouts and writing the data in repodb. This is the case where you deploy zot on an existing layout (including in some of our tests), or you have an older version of zot without repodb and you need to upgrade. In this case the changes to routes.go are not enough.